cmd/put.go

```diff
@@ -175,7 +176,12 @@ func serverMode() bool {
 func handleServerMode(started time.Time) {
 	client, err := newServerClient(serverURL, serverCert)
 	if err != nil {
-		die(err.Error())
+		die("%s", err.Error())
+	}
+
+	_, err = client.StartHeartbeat(heartbeatFreq)
```

Comment:
I think nothing ever uses the return value of this. Either it should be used here to ensure the ticker stops, or if it's definitely unnecessary, the method should be reimplemented to not create and return the first value.

It might make more sense and be preferable to have no public StartHeartbeat function, but instead start the heartbeat when MakingIRODSConnections is called, store the quit channel, then quit during ClosedIRODSConnections().

This way client doesn't have to deal with an implementation detail of getting the IRODS connection tracker to work properly.